### ISSUE TITLE
[JS] Expecting array from tuple result

### ISSUE THREAD

**Issue by rguikers** [eris-ltd/eris-contracts.js#48]
In 0.12 : take contract 
```
contract Test {

    function getAddress() returns (address) {
      return this;
    }

    function getNumber() returns (uint) {
      return 100;
    }

    function getCombination() returns (uint _number, address _address) {
      _number = 100;
      _address = this;
    }

}
```

and code :

```
contractAPI.getCombination(function(err, data) {
  if(err){
    console.log(err);
  } else {
    console.log("getCombination : " + data);
  }
})
```

This returns :
```
getCombination : 00000000000000000000000000000000000000000000000000000000000000640000000000000000000000006A4546A31DD91AD3E5CE01C4E796680A5D32701B
```

I would expect it to return an array, data[0] -> 64, data[1] -> address

**Comment by nottherealbatman** [comment]
Have you tried adding `constant` at the `getCombination` function? see issue #46

**Comment by rguikers** [comment]
Yep, thats it, when using constant the array is returned as expected, and the address is again preceded with '0x'.
New result with contract :
```
contract Test {
    function getAddress() constant returns (address) {
      return this;
    }
    function getNumber() constant returns (uint) {
      return 100;
    }
    function getCombination() constant returns (uint _number, address _address) {
      _number = 100;
      _address = this;
    }
}
```

and code :

```
contractAPI.getAddress(function(err, data) {
  if(err){
    console.log("ERROR");
    console.log(err);
  } else {
    console.log("getAddress : " + data);
  }
})
contractAPI.getNumber(function(err, data) {
  if(err){
    console.log(err);
  } else {
    console.log("getNumber : " + data);
  }
})

contractAPI.getCombination(function(err, data) {
  if(err){
    console.log(err);
  } else {
    console.log("getCombination : " + data[0] + ":" + data[1]);
  }
})
```

returns :
```
getAddress : 046B812E690E4CB54281CC469A087B0E15089EAC
getNumber : 100
getCombination : 100:0x046B812E690E4CB54281CC469A087B0E15089EAC
```
 expected : 
```
getAddress : 046B812E690E4CB54281CC469A087B0E15089EAC
getNumber : 100
getCombination : 100:046B812E690E4CB54281CC469A087B0E15089EAC
```
also without using constant.. Thanks @nottherealbatman

**Comment by NodeGuy** [comment]
Fixed in 0.15.5.
